src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalTypeTestEnhancedInstanceof.java

```diff
@@ -15,30 +15,28 @@
 
 */
 
-//non-compiled with javac: Compilable with Java14
+//non-compiled with javac: Compilable with Java17
 package com.puppycrawl.tools.checkstyle.checks.coding.illegaltype;
 
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.Map;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
Is it compilation problem causing this update? If not let revert, Inputs are weird by design. All extra code is by design.

Reply:
Kind of. I added new types in code and it failed to compile. Star import makes it easier

Reply:
Import in illegalType might be very meaningful, better to avoid `*` please use exact imports to avoid extra questions and verifications.

Reply:
done